src/writer/update-proposal.ts

```diff
@@ -73,7 +74,8 @@ export async function action(body, ipfs): Promise<void> {
     discussion: msg.payload.discussion,
     choices: JSON.stringify(msg.payload.choices),
     scores: JSON.stringify([]),
-    scores_by_strategy: JSON.stringify([])
+    scores_by_strategy: JSON.stringify([]),
+    flagged: +containsFlaggedLinks(msg.proposal.body)
```

Comment:
but it is `msg.payload.body` in line 73 🙈

Reply:
The danger of copilot autocompletion